internal/testplanet/versioning.go

```diff
@@ -59,10 +60,12 @@ func (planet *Planet) NewVersionInfo() version.Info {
 }
 
 // NewVersionConfig returns the Version Config for this planet with tuned metrics.
-func (planet *Planet) NewVersionConfig() version.Config {
-	return version.Config{
-		ServerAddress:  fmt.Sprintf("http://%s/", planet.VersionControl.Addr()),
-		RequestTimeout: time.Second * 15,
-		CheckInterval:  time.Minute * 5,
+func (planet *Planet) NewVersionConfig() checker.Config {
```

Comment:
Looking at this now, I'm honestly not sure. These look equivalent to me.

Reply:
Okay, so the thing about this diff is that we're now embedding the client config which means that in order to write this config as a struct literal we'd have to import the client package which we otherwise don't have to do.

Reply:
:thinking: I don't understand the answer. For me, it looks like it was what you said in the first comment.

Anyway, the result is the same, as you mentioned.

Reply:
The definition of `checker.Config` has changed to have an embedded field. If you use a struct literal here like it was before you'd have to also create a struct literal of that embedded field which, in this case, would mean importing the client package.

Reply:
oh, right! I've got it.
But there is no extra package to import they are both in the same `checker` package.

Anyway, I'm good as it's.

Thanks for the explanation.